libraries/AP_HAL_ChibiOS/hwdef/PH4-mini-bdshot/hwdef.dat

```diff
@@ -0,0 +1,14 @@
+# hw definition file for processing by chibios_hwdef.py
+# for Holybro PH4-mini hardware.
+
+include ../PH4-mini/hwdef.dat
+
+undef PA10 PE11
+undef USART3_RX 
```

Comment:
This is not correct. You don't need to disable the UART entirely, just make sure it doesn't use DMA - put NODMA at the end of the new definition